lib/galaxy/webapps/galaxy/controllers/visualization.py

```diff
@@ -766,13 +767,13 @@ def _handle_plugin_error( self, trans, visualization_name, exception ):
         """
         Log, raise if debugging; log and show html message if not.
         """
-        log.exception( 'error rendering visualization (%s): %s', visualization_name, str( exception ) )
+        log.exception( 'error rendering visualization (%s): %s', visualization_name, unicodify( exception ) )
```

Comment:
```python
log.exception( 'error rendering visualization (%s)', visualization_name )
```
should be enough since the exception info is logged anyway by `log.exception()` .